webpack_loader/loader.py

```diff
@@ -64,7 +67,7 @@ def get_bundle(self, bundle_name):
             timeout = self.config['TIMEOUT'] or 0
             timed_out = False
             start = time.time()
-            while assets['status'] == 'compiling' and not timed_out:
+            while assets['status'] in ['compile', 'compiling'] and not timed_out:
```

Comment:
I'm not an expert in this area, I will suppose you know what you are doing when adding the *compile* status :)

Anyway, I think you should test this status just like you test the *compiling* one. I found occurrences of the compiling status in the following places:

The test_request_blocking func:

https://github.com/gilmrjc/django-webpack-loader/blob/webpack-bundle-tracker-1/tests/app/tests/test_webpack.py#L195

The test_timeouts func:

https://github.com/gilmrjc/django-webpack-loader/blob/webpack-bundle-tracker-1/tests/app/tests/test_webpack.py#L224

Reply:
Actually the `compiling` state was removed in favor of `compile`.

You can see it was removed here:
https://github.com/owais/webpack-bundle-tracker/commit/d1f8e39c0c5bde252c1025174607ebf048ca9f94#diff-6d186b954a58d5bb740f73d84fe39073L60

And changed here:
https://github.com/owais/webpack-bundle-tracker/commit/d1f8e39c0c5bde252c1025174607ebf048ca9f94#diff-6d186b954a58d5bb740f73d84fe39073R102

I will remove the `compiling` reference. Thanks